app/navigation/AppNavigation.tsx

```diff
@@ -147,9 +161,49 @@ export default (): any => {
       resetURL();
     }
   }
+  const apiJsonData = [
+    {
+      apiEndpoint: appConfig.countryGroups,
+      method: 'get',
+      postdata: {},
+      saveinDB: true,

```

Comment:
Use consistent naming conventions (e.g. camelCase) for left attributes.

Reply:
If we change this then need to change for common method and for all API call.

Reply:
Ah it is fine then.